Cargo.toml

```diff
@@ -56,13 +56,16 @@ clap = "2.32.0"
 failure = "0.1.3"
 walkdir = "2.2.7"
 
+[patch.crates-io]
+gimli = { git = "https://github.com/mitsuhiko/gimli", rev = "3aa132cb7c67d5409e433ff82c0d4a3c89bc7229" }
```

Comment:
As discussed offline: Let's make this an explicit dependency and remove the `crates` craft target.